src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = document.querySelector('tbody');
+const rowsForSort = [...tbody.querySelectorAll('tr')];
```

Comment:
Move 'rowsForSort' inside the event listener to ensure sorting is performed on the most up-to-date table rows.